Makefile

```diff
@@ -31,31 +31,23 @@ build/llvm.BUILT:
 		$(LLVM_PROJ_DIR)/llvm
 	ninja -v -C build/llvm \
 		install-clang \
-		install-libclang \
 		install-clang-format \
 		install-clang-tidy \
-		install-clangQuery \
 		install-clang-apply-replacements \
 		install-lld \
-		install-llc \
-		install-llvm-ar \
```

Comment:
For ranlib and strip we have `install-llvm-ranlib` in addition to `install-ranlib`.  Do they do different things?  If you are going to remove the `install-llvm-` variants presumably you should do so for ranlib and strip too?

Reply:
They do different things. I realize this is non-obvious, and it would be way nicer if LLVM just gave us a single install rule instead of doing all this manually. I don't know of such a rule.

Reply:
I found some llvm methods for doing what we want (one of them is called LLVM_INSTALL_TOOLCHAIN_ONLY) but we can do that in a followup.  Happy for this to land as is.